src/ontology/reports/cl-base-diff.md

```diff
@@ -0,0 +1,7183 @@
+# Ontology comparison
+
+## Left
+- Ontology IRI: `http://purl.obolibrary.org/obo/cl/cl-base.owl`
+- Version IRI: `http://purl.obolibrary.org/obo/cl/cl-base/cl/releases/2024-05-15/cl-base.owl/cl/cl-base.owl`
+- Loaded from: `file:/work/src/ontology/tmp/current-base-release.obo`
+
+## Right
+- Ontology IRI: `http://purl.obolibrary.org/obo/cl/cl-base.owl`
+- Version IRI: `http://purl.obolibrary.org/obo/cl/cl-base/cl/releases/2024-07-10/cl-base.owl/cl/cl-base.owl`
+- Loaded from: `file:/work/src/ontology/../../cl-base.obo`
+
+### Ontology imports 
+
+
+
+### Ontology annotations 
+#### Removed
+- [versionInfo](http://www.w3.org/2002/07/owl#versionInfo) "2024-05-15"^^[string](http://www.w3.org/2001/XMLSchema#string) 
+
+- [comment](http://www.w3.org/2000/01/rdf-schema#comment) "See PMID:15693950, PMID:12799354, PMID:20123131, PMID:21208450; Contact Alexander Diehl, addiehl@buffalo.edu, University at Buffalo."^^[string](http://www.w3.org/2001/XMLSchema#string) 
+
+#### Added
+- [versionInfo](http://www.w3.org/2002/07/owl#versionInfo) "2024-07-10"^^[string](http://www.w3.org/2001/XMLSchema#string) 
+
+- [comment](http://www.w3.org/2000/01/rdf-schema#comment) "See PMID:15693950, PMID:12799354, PMID:20123131, PMID:21208450; Contact Alexander Diehl, addiehl@buffalo.edu, university at buffalo."^^[string](http://www.w3.org/2001/XMLSchema#string) 
+
+
+### Ammon's horn basket cell `http://purl.obolibrary.org/obo/CL_2000088`
+#### Removed
+- [Ammon's horn basket cell](http://purl.obolibrary.org/obo/CL_2000088) [contributor](http://purl.org/dc/terms/contributor) [Q35563349](https://www.wikidata.org/entity/Q35563349) 
+
+#### Added
+- [Ammon's horn basket cell](http://purl.obolibrary.org/obo/CL_2000088) [contributor](http://purl.org/dc/terms/contributor) [Q35563349](http://www.wikidata.org/entity/Q35563349) 
+
+
+### B cell `http://purl.obolibrary.org/obo/CL_0000236`
+#### Removed
+- [B cell](http://purl.obolibrary.org/obo/CL_0000236) [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "ZFA:0009142"^^[string](http://www.w3.org/2001/XMLSchema#string) 
+  - [mapping_justification](http://purl.obolibrary.org/obo/mapping_justification) "https://w3id.org/semapv/vocab/UnspecifiedMatching"^^[string](http://www.w3.org/2001/XMLSchema#string) 
+
+#### Added
+- [B cell](http://purl.obolibrary.org/obo/CL_0000236) [database_cross_reference](http://www.geneontology.org/formats/oboInOwl#hasDbXref) "ZFA:0009142"^^[string](http://www.w3.org/2001/XMLSchema#string) 
+  - [mapping_justification](https://w3id.org/sssom/mapping_justification) "https://w3id.org/semapv/vocab/UnspecifiedMatching"^^[string](http://www.w3.org/2001/XMLSchema#string) 
```

Comment:
It might be because the diff is now generated using the OBO format.

Reply:
Looking carefully, I noticed that the change was in the mapping_justification annotation. Previously, it used the wrong curie and then expanded it to OBO instead of sssom.